<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenshotWizardPage</class>
 <widget class="QWizardPage" name="ScreenshotWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Capture whole screen:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="screenComboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="captureWholeScreenPushButton">
         <property name="text">
          <string>Capture</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Capture screen part:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="captureScreenPartPushButton">
       <property name="text">
        <string>Capture</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Capture window:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ActionTools::ChooseWindowPushButton" name="captureWindowPushButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ActionTools::ImageLabel" name="captureImageLabel" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActionTools::ChooseWindowPushButton</class>
   <extends>QPushButton</extends>
   <header>actiontools/choosewindowpushbutton.hpp</header>
  </customwidget>
  <customwidget>
   <class>ActionTools::ImageLabel</class>
   <extends>QWidget</extends>
   <header>actiontools/imagelabel.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
